doc/CMakeLists.txt

```diff
@@ -51,9 +51,16 @@ ENDIF(DEAL_II_COMPONENT_DOCUMENTATION)
 # Always install a minimalistic README and LICENSE file:
 #
 
+# Bloody hack to get an empty relative path:
```

Comment:
I remember there was a reason for this, i.e. why <code>./</code> was specifically not possible to use. But, let's see.